From: Andrew Morton <akpm@linux-foundation.org>
Subject: vmalloc-add-reviewers-for-vmalloc-code-checkpatch-fixes
Date: Tue Oct 18 03:17:35 PM PDT 2022


WARNING: Misordered MAINTAINERS entry - list file patterns in alphabetic order
#39: FILE: MAINTAINERS:13300:
+F:	mm/vmalloc.c
+F:	include/linux/vmalloc.h

total: 0 errors, 1 warnings, 21 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

./patches/vmalloc-add-reviewers-for-vmalloc-code.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: "Uladzislau Rezki (Sony)" <urezki@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 MAINTAINERS |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/MAINTAINERS~vmalloc-add-reviewers-for-vmalloc-code-checkpatch-fixes
+++ a/MAINTAINERS
@@ -13302,8 +13302,8 @@ L:	linux-mm@kvack.org
 S:	Maintained
 W:	http://www.linux-mm.org
 T:	git://git.kernel.org/pub/scm/linux/kernel/git/akpm/mm
-F:	mm/vmalloc.c
 F:	include/linux/vmalloc.h
+F:	mm/vmalloc.c
 
 MEMORY HOT(UN)PLUG
 M:	David Hildenbrand <david@redhat.com>
_
